http/handlers.go

```diff
@@ -121,10 +124,15 @@ func DecodeRequestWithHeaders(_ context.Context, req *http.Request, pathParams m
 }
 
 func getScheme(req *http.Request) string {
-	var xForwardedProtoHeader = req.Header.Get("X-Forwarded-Proto")
-
-	if xForwardedProtoHeader != "" {
-		return xForwardedProtoHeader
+	var forwardedHeader = req.Header.Get("Forwarded")
+
+	if forwardedHeader != "" {
```

Comment:
What about falling back to X-Forwarded-Proto if Forwarded is not provided?

Reply:
Do we decided to not handle X-Forwarded-Proto anymore ? is it safe ?

Reply:
After discussions we ended up with the conclusion that is was safe to drop X-Forwarded-Proto